### ISSUE TITLE
BBB - Video Playback Pausing by students, not shown to teacher, only impacts CLASSROOM playback for students.

### ISSUE THREAD

**Issue by KBellows** [bigbluebutton/bigbluebutton#10682]
When showing a YouTube video through BBB, students have the capability to pause playback, skip around the video, click settings, etc. When the students are doing this, it does so for ALL other students in the class, but NOT the teacher.

Can this option be turned off?

ONLY navigable to the teacher (or presenter), and students aren't able to pause or change settings for the video and have to apply that to ALL students in the class?

**Comment by ffdixon** [comment]
Can you reproduce this error on https://demo.bigbluebutton.org/?

**Comment by KBellows** [comment]
I wasn't able to get into a conference in demo mode above, but I did end up having another teacher add me as a student to reproduce it in real time on another account, for another class. 

As a student, I can pause, skip, change volume, change settings, etc. (Just like I was using YouTube on my own). The problem with this is that it only impacts the video experiences for the students. It does NOT show it for the teacher. I had no idea that the kids were messing with the play back because the video sharing went on as intended for the teacher's experience. It only impacted the view-ability for the students. But the weird this is, it isn't only changing the experience for the student that is messing with the controls. It impacts the experience for all of the other students, no matter who is manipulating the playback.

**Comment by ffdixon** [comment]
Hmm -- can you try again.  We were just able to enter a conference on the demo server (you need to create an account first).

I just tried sharing video with a moderator and two students (A and B).  As student A, I stopped the video, but it only stopped on that session (student B was not affected).

Furthermore, the videos are synced every 5 seconds, so even though as Student A I stopped playing, it immediately started playing again in sync with the instructor.

>As a student, I can pause, skip, change volume, change settings, etc. (Just like I was using YouTube on my own). The problem with this is that it only impacts the video experiences for the students.

Were you the presenter at the time?  The presenter controls the playback of video, and any moderator can make anyone the current presenter.

**Comment by KBellows** [comment]
Hello!  I was able to finally log in and create a demo session (I kept getting a 502 Gateway error.)

I was not able to replicate the issue. BUT, I was able to actually see from the other students' perspectives what was going on. I think what happened here is that kids are silly and were not sure what was going on, and so, the information I provided you was from students who aren't exactly sure how BBB works.

Your explanation in INCREDIBLY helpful, and I will share this with my department and hopefully it clears up some of their frustrations, but assuages some of their concerns about the abilities of the kids to affect others' learning experiences.

Thank you for your time, I am sorry for bothering you! Hopefully, I won't have to bother you again about it!
